sklearn/utils/estimator_checks.py

```diff
@@ -710,7 +710,7 @@ def check_classifiers_one_label(name, Classifier):
 
 
 def check_classifiers_train(name, Classifier):
-    X_m, y_m = make_blobs(random_state=0)
```

Comment:
The test `sklearn.tests.test_common.test_non_meta_estimators` failed for the `AdaBoostClassifier` when using the default 100 samples. @ndawe thought that there were too few samples for the classifier to work stably, so I increased to 1000 samples. When I timed the tests, I didn't see an appreciable increase in run time.

I went back and timed the `sklearn.tests.test_common.test_non_meta_estimators` tests at 100, 300, and 1000 samples. It seems that changing to 1000 samples increases the mean time of those tests by ~1-2 seconds. I reduced to 300 samples, which also allows the `AdaBoostClassifier` to find an adequate solution, but doesn't increase execution time of the tests.